src/components/ManaCalculator/components/OutputForm.tsx

```diff
@@ -5,48 +5,39 @@ import humanizeDuration from 'humanize-duration';
 
 export function OutputForm() {
   const { state } = useManaState();
-  const {
-    generatedRewards,
-    passiveRewards,
-    totalTPS,
-    msToTransaction,
-    passiveMsToTransaction,
-  } = useResults(state);
+  const results = useResults(state);
+  const passiveRewards = fromMicro(results.passiveRewards).toFixed(2);
+  const manaGenerated = fromMicro(results.generatedRewards).toFixed(2);
+  const totalTPS = results.totalTPS.toFixed(2);
+  const humanizer = humanizeDuration.humanizer({
+    units: ['y', 'mo', 'w', 'd', 'h', 'm', 's', 'ms'],
+    maxDecimalPoints: 3,
+  });
```

Comment:
Wondering what the decimals should be.
As the minimum amount of mana is micromana, 6 places after the decimals could theoretically make sense?
TPS is fine.
For the time I just picked 3 basically randomly which gives us a micro seconds resolution

Reply:
Any opinions on that? @begonaalvarezd @marc2332 ?

Reply:
Solved in https://github.com/iotaledger/iota-wiki/pull/1324